README.md

```diff
@@ -90,6 +92,7 @@ scdl me -f
     --onlymp3                   Download only the streamable mp3 file, even if track has a Downloadable file
     --path [path]               Use a custom path for downloaded files
     --remove                    Remove any files not downloaded from execution
+    --write-cover               Download track cover art of tracks as seperate image file
```

Comment:
I think this needs to be removed from this PR since it should just pertain to downloading a set number of tracks.